components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -247,11 +247,6 @@ func (w *NotebookWebhook) Handle(ctx context.Context, req admission.Request) adm
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
 
-		// Only Mount ca bundle on new notebook creation
-		err = CheckAndMountCACertBundle(ctx, w.Client, notebook, log)
-		if err != nil {
-			return admission.Errored(http.StatusInternalServerError, err)
-		}
 	}
 
 	// Check Imagestream Info both on create and update operations
```

Comment:
very good idea to do that cert check and mount on both events, create and update

Reply:
Thanks :+1: